main.tf

```
@@ -129,6 +129,7 @@ module "ecs_alb_service_task" {
   health_check_grace_period_seconds = var.health_check_grace_period_seconds
   task_cpu                          = coalesce(var.task_cpu, var.container_cpu)
   task_memory                       = coalesce(var.task_memory, var.container_memory)
+  ignore_changes_task_definition    = var.task_ignore_changes
```

```suggestion
  ignore_changes_task_definition    = var. ignore_changes_task_definition
```

We can use the same name to avoid name explosion :)

---

Fixed in latest rev